app/src/main/res/values/strings.xml

```diff
@@ -111,5 +111,12 @@
     <string name="symbol_percent"> %</string>
     <string name="title_post_detail">%1$s Post</string>
     <string name="title_selected_image_deital">Image detail</string>
-    <string name="description_write_a_comment">Write a comment...</string>
+    <string name="description_write_a_comment">Write a comment &#8230;</string>
+    <string name="title_no_media">00:00</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_no_media` appears to be unused
 Unused resources make applications larger and slow down builds.